src/unix/udp.c

```diff
@@ -433,6 +433,8 @@ int uv__udp_send(uv_udp_send_t* req,
 
   if (empty_queue && !(handle->flags & UV_UDP_PROCESSING)) {
     uv__udp_sendmsg(handle);
+    if (!QUEUE_EMPTY(&handle->write_queue))
```

Comment:
@indutny can you add a comment here please? Something like "If the request was queued we need to poll for io"